addons/arsenal/missions/Arsenal.VR/initPlayerLocal.sqf

```diff
@@ -60,11 +58,11 @@ for "_i" from 0 to 1 do {
     _target setSpeedMode "LIMITED";
 };
 
-//--- Armored vehicles
+// Armored vehicles to the rear of the player
 private _vehicles = [];
 
 private _step = 15;
-_position = [position _unit select 0,(position _unit select 1) + 30,0];
+_position = _unit getRelPos [30, 180];
```

Comment:
Not the same thing, unit may rotate (though probably at this step it doesn't have the chance to rotate yet?)

Reply:
> Not the same thing, unit may rotate (though probably at this step it doesn't have the chance to rotate yet?)

I know it's not the same thing, but I figured it's cleaner and that the player couldn't have moved when this was executed, as the same structure is used multiple times further above in the code (e.g. `_position = _unit getRelPos [20, 90];`).